streaming/src/main/scala/org/apache/gearpump/streaming/StreamApplication.scala

```diff
@@ -74,5 +82,6 @@ object StreamApplication {
   }
 
   val DAG = "DAG"
-}
 
+  private val defaultPartitioner = new HashPartitioner()
```

Comment:
Can we make HashParitioner as global default partitioner? But, make every StreamApplication able to override it with its own partitioner as app-wide default partitioner?